saskatoon/harvest/forms.py

```diff
@@ -459,7 +503,8 @@ def clean_end_date(self):
         ]:
             raise forms.ValidationError(
                 _(
-                    'Harvests cannot be scheduled over multiple days: start and end dates must match.'
+                    'Harvests cannot be scheduled over multiple days: '
+                    'start and end dates must match.'
```

Comment:
```suggestion
                    "Harvests cannot be scheduled over multiple days: "
                    "start and end dates must match."
```
Also please remove the `# noqa: E501` if no longer needed